### DISCUSSION TITLE
How to map celery queues to solids?

### DISCUSSION THREAD

**Discussion by lgblkb** [dagster-io/dagster#4349] [Q&A] [answered]
I'm using celery executor to run my pipelines. I've created several celery workers that listen to different queues, but I could not find a way to assign specific celery queue for specific solid in a solid's `run_config`. Any help is appreciated!)

**Comment by alangenfeld** [comment, marked as answer]
There is not a way to do it via `run_config`, but you can set the `tag` on your solid https://docs.dagster.io/deployment/guides/kubernetes/customizing-your-deployment#configuring-celery-queues

**Reply by lgblkb** [reply]
That's exactly what I was looking for! Thank you!
